scripts/devenv-builder/manage-vm.sh

```diff
@@ -161,7 +162,7 @@ function action_help {
 }
 
 function usage {
-    script_name=$(basename $0)
+    script_name=$(basename "$0")
```

Comment:
Let's use local -r in the declaration, or get rid of this variable completely.

Reply:
I switched it to `local -r`, thinking having the `script_name` is clear indication on the intention so it's not confusing someone in the future.